x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/text_structure/test_grok_pattern.yml

```
@@ -0,0 +1,42 @@
+"Grok pattern with two named fields":
+  - do:
+      text_structure.test_grok_pattern:
+        body:
+          grok_pattern: "%{WORD}.*%{WORD:first_name} %{WORD:last_name}!"
+          text:
+            - "Hello   Dave Roberts!"
+            - "this does not match"
+  - match:
+      matches:
+        - matched: true
+          fields:
+            first_name:
+              - match: Dave
+                offset: 8
+                length: 4
+            last_name:
+              - match: Roberts
+                offset: 13
+                length: 7
+        - matched: false
+---
+"Grok pattern with two identically named fields":
+  - do:
+      text_structure.test_grok_pattern:
+        body:
```

The language clients use these tests to ensure that the generated clients work with a real Elasticsearch server. Therefore it's good if the inputs and outputs cover all the possible options. I know our current tests don't for many existing APIs, but in this case it should be easy to send one possible value of the `ecs_compatibility` field, because I don't think it will make any difference to the outputs in these particular tests.

```suggestion
        ecs_compatibility: v1
        body:
```

---

fixed